WebUI/electron/main.ts

```diff
@@ -509,7 +487,7 @@ function initEventHandle() {
         if(serviceName == "comfyui-backend") {
             //side effect of relying on ai-backend python env
             appLogger.info(`Starting setup of ${serviceName}`, 'electron-backend')
-            if (!serviceRegistry.getService('ai-backend')?.is_set_up()) {
+            if (!serviceRegistry.getService('ai-backend')?.isSetUp) {
```

Comment:
I don't think, this is still required, as comfyUI was extracted to run in a different python env